depinject/appconfig/module.go

```diff
@@ -19,10 +20,15 @@ var Register = RegisterModule
 // Protobuf message types used for module configuration should define the
 // cosmos.app.v1alpha.module option and must explicitly specify go_package
 // to make debugging easier for users.
-func RegisterModule(msg proto.Message, options ...Option) {
```

Comment:
Why using any of we do a type check/cast just below. Cannot we just use the gogoproto.Message interface?

Reply:
The logic I had was that we may want to support plan old go structs (not protobuf) eventually so one breaking change is better than two. When we discussed in architecture @tac0turtle said gogo is enough, although personally if people are annoyed with too much proto we should lower the barrier

Reply:
We okay to leave as is and merge this?